CHANGELOG.rst

```diff
@@ -1,10 +1,15 @@
 Changelog
 *********
 
+`Unreleased`_ (YYYY-MM-DD)
+--------------------------
+
+- Event callback arguments now always need to provide a `data` argument (major)
```

Comment:
You forgot to mention that the event callbacks now need to return a coroutine, but they don't have to *be* a coroutine, correct?

Reply:
I believe they always had to be a coroutine unless `asyncio.ensure_future(callback(...))` silently discards `None` or other weird return values which I doubt. So, that didn't change and it's now explicit. :slightly_smiling_face: